index.ts

```diff
@@ -450,34 +452,52 @@ export function trait<T extends SuiteTrait | TestTrait>(arg: T): T {
 function createNumericBuiltinTrait(traitSymbol: any, fn: (ctx: Mocha.ISuiteCallbackContext, value: number) => void) {
     return function(value: number): MethodDecorator & PropertyDecorator & ClassDecorator & SuiteTrait & TestTrait {
         return trait(function() {
+            // TODO: Implement an overload selector similar to the testOverload function
+
             if (arguments.length === 1) {
+                // Class decorator
                 const target = arguments[0];
                 target[traitSymbol] = value;
-            } else if (arguments.length === 2 && typeof arguments[1] === "string") {
+                return;
+            }
+
+            /* istanbul ignore if  */
+            if (arguments.length === 2 && typeof arguments[1] === "string") {
+                // PropertyDecorator, some TSC versions generated property decorators when decorating method
```

Comment:
Is this not a static property decorator?

Reply:
typo here? `istanbul ignore if`? or do we have unused code?

Reply:
I think the old TSC was generating these in places where I expected method decorators.
To me this looks like dead code but I am hesitant to delete it.
I don't have time to invest in installing array of TSC versions and test agains each of them (similar to typings tests).

Reply:
okay